Makefile

```diff
@@ -213,8 +213,8 @@ clippy: pre-clippy
 		-A clippy::implicit_hasher -A clippy::large_enum_variant -A clippy::new_without_default \
 		-A clippy::neg_cmp_op_on_partial_ord -A clippy::too_many_arguments \
 		-A clippy::excessive_precision -A clippy::collapsible_if -A clippy::blacklisted_name \
-		-A clippy::needless_range_loop -D rust-2018-idioms -A clippy::redundant_closure \
```

Comment:
Denying rust-2018-idioms can lead to compilation of kvproto, and it can't be fixed by explicitly allow on kvproto's source. Do you have any ideas? /cc @nrc

Reply:
I think adding `#[allow(bare_trait_objects)]` to `lib.rs` in kvproto should fix the errors. There is a bug in Clippy where that is ignored for path deps, but with git deps it should be ok (https://github.com/rust-lang/rust-clippy/issues/1066). If that doesn't work, then I think just removing the deny here is fine - it's not a correctness issue and the idioms are generally easy to fix.

Reply:
> I think adding #[allow(bare_trait_objects)] to lib.rs

That was the first thing I tried, and it didn't work.

Reply:
:-( I guess just remove the lint for now then. It should be fixed fairly soon